frontend/test/metabase/scenarios/smoketest/admin.cy.spec.js

```diff
@@ -18,6 +18,65 @@ describe("metabase-smoketest > admin", () => {
   before(() => restore("blank"));
 
   describe("Admin can setup an account", () => {
+    it("should set up Metabase without first name and last name (metabase#22754)", () => {
```

Comment:
This test is based on the test below it `"should set up Metabase"`. Apart from the two change I commented below, the rest of the test are the same.